corehq/apps/reports/static/reports/javascripts/reports.async.js

```diff
@@ -160,7 +136,10 @@ var HQAsyncReport = function (o) {
                     self.hqLoading.fadeIn();
                 }
 
-            }
+            },
+            complete: function() {
+                self.standardReport.filterSubmitButton.button('reset');
```

Comment:
this is needed for an async report to re-enable the apply button. there's some complicated logic above, but i don't think it's necessary. `reset` keeps all the styles from what i noticed (perhaps fixed in some version upgrade)